ERCS/erc-trustless-agents.md

```diff
@@ -0,0 +1,216 @@
+# ERC-XXXX Trustless Agents v0.4
+
+\---  
+title: Trustless Agents  
+description: A standard interface to discover agents and establish trust through reputation and validation  
+author: Marco De Rossi <marco@metamask.io>, Davide Crapis <davide@ethereum.org>
+discussions-to: \<URL\>
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR. https://ethereum-magicians.org/c/ercs/57